shared/model/vertex_ai_indexes.yaml

```diff
@@ -1,38 +1,23 @@
 dc-all-minilm-l6-v2:
-  type: EMBEDDING
   project_id: "datcom-website-dev"
   location: "us-central1"
-  prediction_endpoint_id: "8518340991868993536"
   index_endpoint_root: "302175072.us-central1-496370955550.vdb.vertexai.goog"
   index_endpoint: "projects/496370955550/locations/us-central1/indexEndpoints/8500794985312944128"
   index_id: "dc_all_minilm_l6_v2_ft_1709655496660"
+  model: "dc-all-minilm-l6-v2-model"
```

Comment:
why not put this in deploy/nl/embeddings.yaml so we can see all embeddings in one place?

Reply:
from my udnerstanding, right now vertex_ai indexes are only used by nl eval tools & this pr is to add support for vertex_ai models in nl server